client/signup/steps/site-type/style.scss

```diff
@@ -50,6 +50,13 @@
 		color: var( --color-text-subtle );
 	}
 
+	.site-type__option-badge {
+		font-size: 12px;
+		margin-top: 6px;
+		// Offset .badge padding-left
+		margin-left: -11px;
```

Comment:
I'm wondering how we can keep this number in sync with the number in `badge/style.scss`.

If the intent is to align by text maybe we could add a `alignByText` prop to `<Badge>` so all the magic numbers are contained within the component.

Or maybe we can just use a shared CSS variable, is it common to create new variables when needed in this project?

Reply:
I had the same thought last night. Normally I'd add a scss configuration variable that both stylesheets can import, but I haven't looked closely at how we handle this in Calypso--I'll investigate.

Reply:
I've moved badge padding to a shared variable in the most recent push. `assets/stylesheets/shared/_variables.scss` is used sparsely, but seems to be the best place for this.

@simison and @jsnajdr ... this related discussion came up in my search on the issue of shared css variables, pa9srA-1m-p2. Is there a current best practice for this, other than using `assets/stylesheets/shared/_variables.scss` for values that should be shared among components?

Reply:
cc all of @Automattic/team-calypso for the latest news on that front :-)

Reply:
Nice. Thanks for the link 🙂